pkg/library/dynaload.go

```diff
@@ -17,14 +18,17 @@ import (
 var skipLibraryPath = os.Getenv("LOCALAI_SKIP_LIBRARY_PATH") != ""
 
 // LoadExtractedLibs loads the extracted libraries from the asset dir
-func LoadExtractedLibs(dir string) {
-	if skipLibraryPath {
-		return
+func LoadExtractedLibs(dir string) error {
+	// Skip this if LOCALAI_SKIP_LIBRARY_PATH is set
+	if os.Getenv("LOCALAI_SKIP_LIBRARY_PATH") != "" {
```

Comment:
mmm this looks a leftover during rebase?

Reply:
I'll double check that nothing stupid happened in the process - but this is an area where I made an intentional change - Specifically, we were swallowing the error returned from os.Setenv() - so this function and the one below were modified to return `error` instead of nothing.

Reply:
Yep, cleanup inbound :)